src/main/java/examples/pingpong/grpc/PingPongGrpcMain.java

```diff
@@ -19,12 +19,16 @@
 import com.daml.ledger.api.v1.PackageServiceOuterClass.GetPackageResponse;
 import com.daml.ledger.api.v1.PackageServiceOuterClass.ListPackagesRequest;
 import com.daml.ledger.api.v1.PackageServiceOuterClass.ListPackagesResponse;
+import com.daml.ledger.api.v1.admin.UserManagementServiceGrpc;
+import com.daml.ledger.api.v1.admin.UserManagementServiceGrpc.UserManagementServiceBlockingStub;
+import com.daml.ledger.api.v1.admin.UserManagementServiceOuterClass.GetUserRequest;
+import com.daml.ledger.api.v1.admin.UserManagementServiceOuterClass.GetUserResponse;
 import com.daml.ledger.api.v1.ValueOuterClass.Identifier;
 import com.daml.ledger.api.v1.ValueOuterClass.Record;
 import com.daml.ledger.api.v1.ValueOuterClass.RecordField;
 import com.daml.ledger.api.v1.ValueOuterClass.Value;
-import com.digitalasset.daml_lf_1_8.DamlLf;
-import com.digitalasset.daml_lf_1_8.DamlLf1;
+import com.daml.daml_lf_1_14.DamlLf;
```

Comment:
can you drop `detectPingPongPackageId` and therefore also the dep on this? it looks like you get the package id via the cli arg now.

Reply:
yeah I noticed that one too and did that already just now locally :D